anatomy

```diff
@@ -757,17 +741,14 @@ same target that the original makefile was invoked with. We follow this up with
 the specific definition for FRC. Because this does not actually generate
 anything, we can remain confident that it will always cause the `SUBDIRS` target
 to be out of date and thus cause us to run again. To wrap everything up, we do
-one last include for `Makefile.targ`. This includes some of the various
-definitions that we use such as `lint_PROG`.
+one last include for `Makefile.targ`.
 
 #### Adding Multiple Source Files
 
 Now that we're this far, adopting this setup to have multiple source files is
 actually quite simple. There are two changes that we need to make to our
 `Makefile.com` file. The first is to append the new object file to the `OBJS`
-variable. The second is to change how we're handling the `lint` target.
-Currently the `lint` target is set to `lint_PROG`. We need to change that to
-`lint_SRCS`. While the `SRCS` assignment we made in `Makefile.com` earlier may
+variable. to While the `SRCS` assignment we made in `Makefile.com` earlier may
```

Comment:
Weirdness here, not sure what happened

Reply:
Leftovers from attempting to deal with SRCS which is no longer really required I think.